### PR TITLE
chore: warns of transactions bug

### PR DESCRIPTION
**Pull request by thiagotnunes**
Happening in versions >= 3.0.0 and <= 3.3.1.

### CODE REVIEW

**Review by skuruppu — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,15 @@
 
 ### [3.3.1](https://www.github.com/googleapis/java-spanner/compare/v3.3.0...v3.3.1) (2021-01-14)
 
+### ⚠ IMPORTANT: Known issue with this version of the client
+
+Since [v3.0.0](https://github.com/googleapis/java-spanner/releases/tag/v3.0.0), users might experience transaction getting stuck in a very rare occasion, where the Spanner backend returns a retryable error when consuming the first record of a read / query in a transaction.
```
Comment (elharo) [inline CHANGELOG.md:20]: "users might experience transaction getting stuck in a very rare occasion, where" --> transactions can get stuck if

File: CHANGELOG.md [hunk 2]
```diff
@@ -31,6 +40,15 @@
 
 ## [3.3.0](https://www.github.com/googleapis/java-spanner/compare/v3.2.1...v3.3.0) (2021-01-07)
 
+### ⚠ IMPORTANT: Known issue with this version of the client
+
+Since [v3.0.0](https://github.com/googleapis/java-spanner/releases/tag/v3.0.0), users might experience transaction getting stuck in a very rare occasion, where the Spanner backend returns a retryable error when consuming the first record of a read / query in a transaction.
```
Comment (elharo) [inline CHANGELOG.md:45]: ditto

File: CHANGELOG.md (hunk 1 above)
Comment (thiagotnunes) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (thiagotnunes) [reply CHANGELOG.md]: Done.

**Pull request merged**